test/system/projects/workflow_executions_test.rb

```diff
@@ -455,5 +457,136 @@ class WorkflowExecutionsTest < ApplicationSystemTestCase
       assert_no_link I18n.t(:'workflow_executions.show.edit_button')
       assert_no_link I18n.t(:'workflow_executions.show.remove_button')
     end
+
+    test 'can successfully delete multiple workflows at once' do
+      visit namespace_project_workflow_executions_path(@namespace, @project)
+
+      assert_selector 'h1', text: I18n.t(:'workflow_executions.index.title')
+
+      assert_text 'Displaying 12 items'
```

Comment:
Change the `Displaying n items` to use the viral pagy limit_component summary translation throughout this file